app/controllers/application_controller.rb

```diff
@@ -14,7 +14,12 @@ class ApplicationController < ActionController::Base
   # redirected to the specified redirect URI in this request. In this example that is `identity_callback` seen below.
   def authenticate_with_identity
     identity_state = SecureRandom.hex
-    cookies.encrypted[:identity_state] = identity_state
+    cookies.encrypted[:identity_state] = {
+      value: identity_state,
+      same_site: "None",
+      secure: true,
```

Comment:
Could we do this in the secure headers file similar how you did it in Manage?

Reply:
You mean setting a blanket rule in the secure headers file so all cookies are same site none? 
https://github.com/github/secure_headers/blob/main/docs/cookies.md

That works for Switchboard, my only concern was it may not be wise for partners to do a blanket rule. Instead they should choose which ones to make same_site individually for the embedded app to work.

Reply:
Understandable about limiting this to specific cookies. I think this approach works and shows app partners how to limit to each cookie.